pinot-common/src/main/java/com/linkedin/pinot/pql/parsers/pql2/ast/ComparisonPredicateAstNode.java

```diff
@@ -65,60 +85,115 @@ public String toString() {
         '}';
   }
 
+  /**
+   * This function creates a standard unified shaped string for the value and operand side of the comparison
+   * @return
+   */
+  private String CreateStringForComparison(){
```

Comment:
I added this function since same thing is needed for building FilterQueryTree and HavingQueryTree